### ISSUE TITLE
Compile fails via Carthage with Xcode 6.3 / iOS 8.3 "Unsupported architecture"

### ISSUE THREAD

**Issue by pizthewiz** [daltoniam/SwiftHTTP#83]
Possibly related to #76?

I'm not entirely sure what is going on here, it seems that the entire build is blowing up with a bad architecture, as if it were forcing `arm64` with the simulator or something:
```sh
$  carthage update
*** Fetching SwiftHTTP
*** Checking out SwiftHTTP at "0.9.4"
*** xcodebuild output can be found in /var/folders/fk/y_3v6btd0s911nzy83hf6s8h0000gn/T/carthage-xcodebuild.Upse7i.log
*** Building scheme "SwiftHTTP" in SwiftHTTP.xcodeproj
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/usr/include/sys/cdefs.h:680:2: error: Unsupported architecture
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/usr/include/machine/_types.h:34:2: error: architecture not supported
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/usr/include/sys/_types.h:55:9: error: unknown type name '__int64_t'; did you mean '__int128_t'?
...
<unknown>:0: error: too many errors emitted, stopping now
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/System/Library/Frameworks/CoreFoundation.framework/Headers/CoreFoundation.h:11:10: error: could not build module 'Darwin'
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/System/Library/Frameworks/Foundation.framework/Headers/Foundation.h:6:10: error: could not build module 'CoreFoundation'
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/usr/include/objc/objc.h:31:10: error: could not build module 'Darwin'
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/usr/include/mach-o/arch.h:32:10: error: could not build module 'Darwin'
/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator8.3.sdk/usr/include/dispatch/dispatch.h:25:12015-04-20 17:35:59.051 xcodebuild[54181:2009948] [MT] iPhoneSimulator: SimVerifier returned: Error Domain=NSPOSIXErrorDomain Code=53 "Simulator verification failed." UserInfo=0x7fb4961811e0 {NSLocalizedFailureReason=A connection to the simulator verification service could not be established., NSLocalizedRecoverySuggestion=Ensure that Xcode.app is installed on a volume with ownership enabled., NSLocalizedDescription=Simulator verification failed.}
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal arm64 /Users/piz/sandbox/Pied Piper/Carthage/Checkouts/SwiftHTTP/HTTPRequestSerializer.swift
	CompileSwift normal arm64 /Users/piz/sandbox/Pied Piper/Carthage/Checkouts/SwiftHTTP/HTTPResponseSerializer.swift
	CompileSwift normal arm64 /Users/piz/sandbox/Pied Piper/Carthage/Checkouts/SwiftHTTP/HTTPTask.swift
	CompileSwift normal arm64 /Users/piz/sandbox/Pied Piper/Carthage/Checkouts/SwiftHTTP/HTTPUpload.swift
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
(5 failures)

```

**Comment by pizthewiz** [comment]
Same issue occurs with Xcode 6.3.1. For what it is worth the two other items in my `Cartfile` build without issue.

**Comment by daltoniam** [comment]
Ok interesting. I believe I already fixed #76 with the `0.9.4` release. I don't personally use Carthage, but I did just test building the framework in my test iOS and OS X projects and the built fine on xCode 6.3. From what I understand, Carthage only requires the schemes of the xcodeproj to be shared, which they are. I came across this thread:

https://github.com/Carthage/Carthage/issues/381.

I might try installing the Xcode Command Line Tools (from https://developer.apple.com/downloads/index.action?name=Xcode) see if that resolves the issue.
